python/ray/workers/default_worker.py

```diff
@@ -91,6 +91,8 @@
     required=True,
     type=int,
     help="the port of the node's metric agent.")
+parser.add_argument(
+    "--tracing-startup-hook", type=str, help="tracing startup hook.")
```

Comment:
tracing is automatically performed via internal kv, no need to pass it in here now.

Reply:
default_worker creates new Nodes, which need the tracing startup hook to be passed in via ray_params which is then put into internal_kv, so i think we do need this here

Reply:
actually i was mistaken, this is not needed so i removed it.